build.sbt

```diff
@@ -1386,17 +1419,28 @@ ThisBuild / dependencyOverrides ++= Seq(
   "org.hamcrest" % "hamcrest-core" % hamcrestVersion,
   "org.objenesis" % "objenesis" % "2.5.1",
   "org.ow2.asm" % "asm" % "5.0.4",
-  "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion,
-  "org.scala-lang.modules" %% "scala-xml" % "1.2.0",
-  "org.scalacheck" %% "scalacheck" % scalacheckVersion,
-  "org.scalactic" %% "scalactic" % scalatestVersion,
-  "org.scalatest" %% "scalatest" % scalatestVersion,
   "org.slf4j" % "slf4j-api" % slf4jVersion,
   "org.slf4j" % "slf4j-log4j12" % slf4jVersion,
   "org.tukaani" % "xz" % "1.8",
-  "org.typelevel" %% "algebra" % algebraVersion,
-  "org.typelevel" %% "cats-core" % catsVersion,
   "org.xerial.snappy" % "snappy-java" % "1.1.4",
   "org.yaml" % "snakeyaml" % "1.12",
-  "com.nrinaudo" %% "kantan.codecs" % kantanCodecsVersion
 )
+
+ThisBuild / dependencyOverrides ++= Seq(
+  "com.fasterxml.jackson.module" %% "jackson-module-scala" % jacksonVersion,
+  "com.propensive" %% "magnolia" % magnoliaVersion,
+  "io.circe" %% "circe-core" % circeVersion,
+  "io.circe" %% "circe-generic" % circeVersion,
+  "io.circe" %% "circe-parser" % circeVersion,
+  "org.scala-lang.modules" %% "scala-xml" % "1.2.0",
+  "com.nrinaudo" %% "kantan.codecs" % kantanCodecsVersion,
+  //"org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion
+).map(_.cross(CrossVersion.for3Use2_13))
+
+ThisBuild / dependencyOverrides ++= Seq(
+  "org.typelevel" %% "algebra" % algebraVersion,
+  "org.typelevel" %% "cats-core" % catsVersion,
+  "org.scalacheck" %% "scalacheck" % scalacheckVersion,
+  "org.scalactic" %% "scalactic" % scalatestVersion,
+  "org.scalatest" %% "scalatest" % scalatestVersion
+)
```

Comment:
I don't know if this is something scalafmt usually complains about, but it seems this change removed the empty line at the end of the file. To make the git diff simpler, I suggest we place it back.